packages/core/test/helpers/ucanto.js

```diff
@@ -137,6 +137,7 @@ export async function getAggregatorServiceServer (serviceProvider, options = {})
 
   const service = mockService({
     aggregate: {
+      // @ts-expect-error todo
```

Comment:
todo?

Reply:
this is the legacy API mock that will go away